docs/getting-started/README.md

```diff
@@ -1 +1,7 @@
-# getting-started/README.md
+# What is a CRM
+
+A Customer Relationship Management, well-known as CRM, is a way to track relationships with customers. Typically, this is done by storing data, such as contact information and other insights about a customer (recent emails, meetings, phone calls, etc.). CRM’s can be found in many forms, but at Cal-ITP, we use a platform named [Hubspot](https://www.hubspot.com/) for our CRM. Within Hubspot are multiple products that allow us to build out our CRM: *service, sales and marketing*. These products and our context for usage are detailed in this guide.
+
+## Purpose
+
+The CRM’s purpose is to increase the effectiveness of Cal-ITP’s relational communication and outreach efforts by establishing transit provider and organization profiles and contact records that detail past and current communication activity such as emails, phone calls, deals (payment pipeline, transit data pipeline, etc.) and support tickets.
```

Comment:
I think we should either include the more detailed Purpose section (file 6) here, or remove this section and have the Purpose page directly after 'What is a CRM'

Reply:
added the full section to "What is a CRM"